serving/docs/lmi/user_guides/chat_input_output_schema.md

```diff
@@ -139,6 +139,30 @@ Example:
 }
 ```
 
+Starting in v0.29.0, we have added experimental support for vision language models. 
+You can specify an image as part of the content when using a vision language model.
+Image data can either be specified as a url, or via a base64 encoding of the image data.
+
+Example:
+
+```
+{
+    "role": "user",
+    "content": [
+        {
+            "type": "text",
+            "text": "What is this an image of?"
+        },
+        {
+            "type": "image_url",
+            "image_url": {
+                "url": "<base64 encoded image data | image url>"
```

Comment:
why b64 encoded data treat as url?

Reply:
no clue, that's just how openai does it.

separately i'm working on multipart support for our default schema, but that's not finished yet.